lib/cli/collect-files.js

```diff
@@ -92,4 +118,10 @@ module.exports = ({
  * @property {string[]} file - List of additional files to include
  * @property {boolean} recursive - Find files recursively
  * @property {boolean} sort - Sort test files
+ * @typedef {Object} UnmatchedFile - Diagnostic object containing unmatched files
```

Comment:
[Docs] I believe it's standard to have a separate JSDoc tag for each object declaration?

```suggestion
 */
 
/**
 * @typedef {Object} UnmatchedFile - Diagnostic object containing unmatched files
```

Reply:
TIL, thanks! Made changes to have separate tags for each object declaration.